src/App.css

```diff
@@ -34,6 +34,43 @@
   margin-bottom: 1rem;
 }
 
+.nav-btn.back-button,
+.back-continue-buttons .back-button,
+.prev-continue-results-buttons .back-button,
+.prev-save-continue-buttons .back-button {
+  font-family: 'Open Sans';
+  color: #41528c;
+  background-color: transparent;
+  font-weight: 700;
+}
+
+.nav-btn.back-button:hover,
+.back-continue-buttons .back-button:hover,
+.prev-continue-results-buttons .back-button:hover,
+.prev-save-continue-buttons .back-button:hover {
+  background-color: transparent;
+}
+
+.nav-btn.continue-button,
+.back-continue-buttons .continue-button,
+.prev-continue-results-buttons .continue-button,
+.prev-save-continue-buttons .continue-button {
+  font-family: 'Open Sans';
+  background-color: #293457;
```

Comment:
Update these colors when https://github.com/Gary-Community-Ventures/benefits-calculator/issues/765 is completed.